doc/nrf/getting_started.rst

```diff
@@ -18,6 +18,7 @@ After that, pick a sample that is related to the application you want to create
    gs_ins_os
    gs_assistant
    gs_installing
+   gs_recommended_versions
```

Comment:
Instead of including it here directly, I think this should be in the "Installing the SDK manually" section. Because anyone not going into that section should not be exposed to this at all.

Reply:
I don't agree, as long as we still have the Getting Started Assistant (and we still have that one for Linux afaik). That one installs the latest available versions, and thus users might need to know the tested versions.

Reply:
@carlescufi  I see your point, this is especially useful for users installing manually.

But, currently we are not documenting anywhere, which tools and their versions are being installed when using the toolchain manager, and because of that, I also feel this page has value for users that are installing using toolchain manager.

They might have less interest (compared to a user doing manual install), but some users might still need to document what tools they use for building software.

For time being, this is the only place Toolchain Manager users can find the versions for the tools being installed.
(The only other alternative is to do a `<program> --version` on each tool AFTER the installation)

So even though I see your point, then after some more thinking, then having an independent section that can be linked from both manual and `gs_assistent` has value.